xblockmufi/xblockmufi.py

```diff
@@ -72,12 +76,25 @@ def workbench_scenarios():
     """
     Main functions
     """
-    def student_view(self, context=None):
+    def student_view(self, context={}):
         """
         Build the fragment for the default student view
         """
+        context.update(
+            {
+                'display_name': self.display_name,
+                'student_answer': self.student_answer,
+                'is_past_due': self.is_past_due(),
+				'submit_class': self._get_submit_class(),
```

Comment:
fix indent

Reply:
rename var to match value in HTML template.

Reply:
Also, let's rename the function to something more meaningful, for example: `_get_button_display_class()`.